mllib/src/test/java/org/apache/spark/mllib/recommendation/JavaALSSuite.java

```diff
@@ -48,14 +47,14 @@ public void tearDown() {
     sc = null;
   }
 
-  void validatePrediction(
+  private void validatePrediction(
       MatrixFactorizationModel model,
       int users,
       int products,
-      DoubleMatrix trueRatings,
+      double[] trueRatings,
```

Comment:
For the record, this is not a `DenseMatrix<Double>` because the Scala method this relies on returns a `DenseMatrix<Object>` as far as Java is concerned. It ended up being easier just to make this private test method pass around an array, I think.